scripts/covid19_india/cases_count_districts_data/COVID19_cases_indian_districts.tmcf

```diff
@@ -0,0 +1,30 @@
+Node: E:COVID19_cases_india->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_MedicalTest_ConditionCOVID_19_Positive
```

Comment:
If these are cumulative counts, per README, I think the three stat-vars should be:
* CumulativeCount_MedicalTest_ConditionCOVID_19_Positive
* CumulativeCount_MedicalConditionIncident_COVID_19_PatientRecovered
* CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased

Reply:
These are daily counts. Not cumulative counts. The README is wrong. I corrected it now.